i18n/en.json

```diff
@@ -554,6 +554,9 @@
   "admin.customization.gfycatApiSecretDescription": "The API secret generated by Gfycat for your API key. When blank, uses the default API secret provided by Gfycat.",
   "admin.customization.iosAppDownloadLinkDesc": "Add a link to download the iOS app. Users who access the site on a mobile web browser will be prompted with a page giving them the option to download the app. Leave this field blank to prevent the page from appearing.",
   "admin.customization.iosAppDownloadLinkTitle": "iOS App Download Link:",
+  "admin.customization.restrictLinkPreviewsDesc": "Link previews and image link previews will not be shown for the above list of comma-separated domains.",
+  "admin.customization.restrictLinkPreviewsExample": "E.g.: \"corp.mattermost.com, mattermost.org\"",
```

Comment:
@esethna Any suggestions for what we might be able to use as example text? We probably don't want to suggest people block us 😛

Reply:
another possible example: `internal.mycompany.com`

Reply:
That might be a good idea, although we prevent link previews from being generated for local addresses by default, so I wouldn't want to necessarily suggest that we don't do that

Reply:
@berkeka can we update the placeholder text to: `internal.mycompany.com, images.example.com`